### PR TITLE
ChoiceGroupOption: pass original renderer to on render label

### LINKED ISSUE #11575: Pass the original renderer to onRenderLabel in ChoiceGroupOption
#### Describe the feature that you would like added

Currently, the ChoiceGroup component allows you to pass "onRenderLabel" property that is a function used to render the labels instead of the fabric function. This function currently is provided with the original properties but not the original renderer.
If we want to keep the original design but wrap it with another functionality/design - we will have to generate it on our own and even give it an internal class name (ms-ChoiceFieldLabel).

The expected behavior is a 2nd parameter sent to onRenderLabel that is the original renderer (_onRenderLabel) so you will be able to call it and wrap the original rendered label.

#### What component or utility would this be added to

ChoiceGroup / ChoiceGroupOption

#### Have you discussed this feature with our team, and if so, who

No, but this functionality is present in many onRender* functions in fabric. The best example for this is "**onRenderField**" this specific component (ChoiceGroupOption)

FYI @maelgami

### PR DESCRIPTION
**Pull request by dagoltz**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #11575 
- [x] Include a change request file using `$ yarn change`

#### Description of changes

Pass the defaultRender as a second parameter to onRenderLabel. Also added tests to cover the existing and new usages of onRenderLabel.

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/11581)

### CODE REVIEW

**Comment by dagoltz** [comment]
FYI @jdhuntington

**Comment by jdhuntington** [comment]
Awesome; thank you!

File: packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.test.tsx [hunk 1]
```diff
@@ -18,6 +19,33 @@ describe('ChoiceGroupOption', () => {
         <ChoiceGroupOption iconProps={{ iconName: 'CalendarWeek' }} key="G" text="Option G" focused />
         <ChoiceGroupOption iconProps={{ iconName: 'CalendarDay' }} key="H" text="Option H" checked />
         <ChoiceGroupOption iconProps={{ iconName: 'CalendarWeek' }} key="I" text="Option I" disabled />
+        <ChoiceGroupOption
```
Comment (ecraig12345) [inline packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.test.tsx]: Might be better to put this in its own test, like "custom renders correctly" (and then update snapshots again)

File: packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.test.tsx [hunk 2]
```diff
@@ -18,6 +19,33 @@ describe('ChoiceGroupOption', () => {
...
+          key="J"
+          text="Option J"
+          onRenderLabel={(option: IChoiceGroupOptionProps): JSX.Element => {
```
Comment (ecraig12345) [inline packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.test.tsx]: To get the build to pass you'll need to add a comment line `// tslint:disable:jsx-no-lambda` (either around here or near the top of the file). That lint rule generally doesn't matter for tests, so disabling it is fine.

**Comment by dagoltz** [comment]
Thanks, @ecraig12345, I have resolved all issues - waiting for the build.
Who has the right to merge this PR once it is re-approved (if needed)?

**Pull request merged**